Remotion/Web/Development.WebTesting/Configuration/IWebTestSettings.cs

```diff
@@ -0,0 +1,138 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using OpenQA.Selenium;
+using Remotion.Web.Development.WebTesting.DownloadInfrastructure;
+
+namespace Remotion.Web.Development.WebTesting.Configuration
+{
+  /// <summary>
+  /// Represents WebTesting specific settings.
+  /// </summary>
+  public interface IWebTestSettings
+  {
+    /// <summary>
+    /// Browser in which the web tests are run.
+    /// </summary>
+    string BrowserName { get; }
+
+    /// <summary>
+    /// Specifies how long the Coypu engine should maximally search for a web element or try to interact with a web element before it fails.
+    /// </summary>
+    TimeSpan SearchTimeout { get; }
+
+    /// <summary>
+    /// Specifies how long Selenium should maximally wait for issued commands before failing. Default is 1 minute.
+    /// </summary>
+    TimeSpan CommandTimeout { get; }
+
+    /// <summary>
+    /// Specifies how long the <see cref="DownloadHelperBase"/> should wait before looking for the downloaded file. Default is 10 seconds.
+    /// </summary>
+    TimeSpan DownloadStartedTimeout { get; }
+
+    /// <summary>
+    /// Specifies how long the <see cref="DownloadHelperBase"/> should wait for a downloaded partial file to update. Default is 10 seconds.
+    /// </summary>
+    TimeSpan DownloadUpdatedTimeout { get; }
+
+    /// <summary>
+    /// Specifies how long the <see cref="WebTestSetUpFixtureHelper"/> should wait for the WebApplication to return a 200 on <see cref="WebApplicationRoot"/>. Default is 1 minute.
+    /// </summary>
+    TimeSpan VerifyWebApplicationStartedTimeout { get; }
+
+    /// <summary>
+    /// Whenever the element to be interacted with is not ready, visible or otherwise not present, the Coypu engine automatically retries the action
+    /// after the given <see cref="RetryInterval"/> until the <see cref="SearchTimeout"/> has been reached.
+    /// </summary>
+    TimeSpan RetryInterval { get; }
```

Comment:
Default please

Reply:
`RetryInterval` is a required configuration value and therefore has no default value.